### PR TITLE
Add Swedish country lead job ad

### PR DESCRIPTION
**Pull request by greysteil**
For discussion. Copy is the same as Spain, with substitutions.

@hirokitakeuchi @adeclosset @harrison @nqureshi - what do you think?

### CODE REVIEW

**Comment by greysteil** [comment]
My thinking is that this will put the fear of god into SlimPay etc., and I'd be keen to test the water in Sweden anyway. Appreciate we might want to bump it back as a non-priority market though.

@Jennabrown - should have cc-ed you too.

File: app/pages/about/_country_lead_se.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<h3 class="u-text-heading-light u-text-m u-color-heading">Country Lead: Sweden</h3>
+
+<p class="para">
+  Launch GoCardless in Sweden.
+</p>
+
+<p class="para">
+  We've built a world-class product that brings cutting-edge recurring payments
+  technology to a wide variety of businesses, from sole traders to multinational
+  corporations like Tripadvisor, The Guardian and Box. We've gained great
```
Comment (nqureshi) [inline app/pages/about/_country_lead_se.html]: Should CamelCase TripAdvisor

**Comment by nqureshi** [comment]
Great idea - definitely feels worth putting this up. (No downside).

**Comment by harrison** [comment]
Yes great idea!

**Comment by greysteil** [comment]
OK, looks like people are keen on this. @jesssummerfield @moyni what say you? We'll need to set up the job in greenhouse and update the greenhouse link here.

File: app/pages/about/_country_lead_se.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<p class="para">
+  You're adaptable. You can tailor our voice to local conditions and needs,
+  whether you're working with prospective clients or internal teams.
+</p>
+
+<p class="para">
+  You're a native-level Swedish speaker with experience working in Sweden.
+</p>
+
+<a href="http://boards.greenhouse.io/gocardless/jobs/CHANGE_ME"
```
Comment (greysteil) [inline app/pages/about/_country_lead_se.html]: Don't merge without changing this.

**Comment by Jennabrown** [comment]
Yes keen! @harrison do you think we could have one person covering the Nordics or is local language fluency a 'must have'

**Comment by hirokitakeuchi** [comment]
Great idea!


—
Hiroki Takeuchi+44 759 559 7114

**Comment by harrison** [comment]
@greysteil one person would be fine, would also guess that most medium large business are operating across the nordics.

**Comment by Jennabrown** [comment]
Awesome @moyni should just require one of the nordic languages then?

**Comment by jesssummerfield** [comment]
@greysteil greenhouse link: https://boards.greenhouse.io/gocardless/jobs/61303

**Comment by greysteil** [comment]
A couple of offline conversations on this one:
- Sweden's the most important country for us in the Nordics, so this person should be a Swedish country manager first and foremost (Denmark has messed up DD, Norway is small)
- Decision meeting about putting this job up all happened here on GitHub because it's so similar to our other country-manager positions (no one missed out on anything - sorry for not making that clear)

Jess is going to push this to MoveMeOn and Tyba.

Merging!

**Comment by greysteil** [comment]
Thanks everyone!

**Comment by Jennabrown** [comment]
Makes sense! Excited!

**Pull request merged**